### ISSUE TITLE
Authenticity token?

### ISSUE THREAD

**Issue by BillyParadise** [vergilet/repost#4]
Hi great and useful gem.  Is there a way to set an authenticity token for a request?  I assume this is a pretty common use case:

``` 
def redeem_via_code
  repost('/orders/rc',
    params: {
      code: params[:id]
    }
  )
end

**Comment by vergilet** [comment]
Hi, @BillyParadise !
Thanks for the issue report.

Working on the solution :)

**Comment by vergilet** [comment]
Released version `0.2.9` which supports authenticity_token.
Gem is available on rubygem or on Releases Tab here, on github.
Extended README with description how to deal with `authenticity_token` option.

@BillyParadise, please let me know if it works as you expected. 
Thanks!

**Comment by vergilet** [comment]
Closing it.
